src/main/resources/assets/patchouli/lang/de_DE.lang

```diff
@@ -0,0 +1,93 @@
+patchouli.gui.lexicon.shift_for_recipe=Umschalttaste-klicken für Rezept
+patchouli.gui.lexicon.index=Eintragsverzeichnis
+patchouli.gui.lexicon.index.info=Ein Verzeichnis von jedem Kapitel ist Verfügbar in diesem Buch.$(br2)Du kannst den Verzeichnis durchsuchen oder irgendwelche andere Kategorie, ganz einfach durch das Eintippen der Anfrage. Die Suchleiste wird dann erscheinen um dich zu unterstützen.
+patchouli.gui.lexicon.shapeless=Formlos
+patchouli.gui.lexicon.no_results=Keine Ergebnisse
+patchouli.gui.lexicon.sad=:(
+patchouli.gui.lexicon.locked=(Gesperrt)
+patchouli.gui.lexicon.reloaded=Buch neugeladen. Dauerte %d ms.
+patchouli.gui.lexicon.add_bookmark=Lesezeichen hinzufügen
+patchouli.gui.lexicon.remove_bookmark=(Umschalttaste drücken und klicken zum entfernen)
+patchouli.gui.lexicon.multiblock_bookmark=Visualisierte Struktur
+patchouli.gui.lexicon.relations=Verwandte Kapitel
+patchouli.gui.lexicon.not_anchored=Rechts-klicken auf einen Block um die Struktur zu fixieren
+patchouli.gui.lexicon.structure_complete=Fertig!
+patchouli.gui.lexicon.needs_air=(Lösche Blöcke die rot markiert sind)
+patchouli.gui.lexicon.visualize_letter=V
+patchouli.gui.lexicon.seconds=%ss
+patchouli.gui.lexicon.history=Lese-Verlauf
+patchouli.gui.lexicon.history.info=Dieser Bereich speichert einige der vorherigen Kapiteln die du besucht hast.$(br2)Kapiteln werden automatisch hinzugefügt und entfernt während du das Buch durchblätterst. Willst du aber, dass sie länger referenziert bleiben, kannst du sie als Lesezeichen hinzufügen. $(br2)$(o)Nützlicher Hinweis: Versuche mal die Umschalttaste zu drücken und klicke auf der Kapitel-Schaltfläche!$()
+patchouli.gui.lexicon.toast=Neue Kapitel
+patchouli.gui.lexicon.toast.info=Neue Kapitel Freigeschaltet
+patchouli.gui.lexicon.external_link=(Externer Link)
+patchouli.gui.lexicon.loading_error=Lade Fehler!
+patchouli.gui.lexicon.loading_error_hover=(Schwebe für Info)
+patchouli.gui.lexicon.loading_error_log=Prüfe deinen Protokoll für mehr
+patchouli.gui.lexicon.dev_edition=Schreiber
+patchouli.gui.lexicon.edition_str=%s Edition
+patchouli.gui.lexicon.added_by=Hinzugefügt von %s
+patchouli.gui.lexicon.sneak=Schleiche zum anzeigen
+patchouli.gui.lexicon.view=Zeige Eintrag
+patchouli.gui.lexicon.keybind=Tastenbelegung: %s
+patchouli.gui.lexicon.keybind_missing=Keine solche Tastenbelegung: %s
+patchouli.gui.lexicon.objective=Zielsetzung
+patchouli.gui.lexicon.incomplete=Unvollständig
+patchouli.gui.lexicon.complete=Vollständig!
+patchouli.gui.lexicon.mark_complete=Markierung Vollständig
+patchouli.gui.lexicon.mark_incomplete=Markierung Unvollständig
+
+patchouli.gui.lexicon.button.prev_page=Vorher
+patchouli.gui.lexicon.button.next_page=Nächste
+patchouli.gui.lexicon.button.back=Zurück
+patchouli.gui.lexicon.button.back.info=Umschalttaste drücken und klicken um nach Hause zurückzukehren
```

Comment:
... now this _really_ sounds like google translate.
```suggestion
patchouli.gui.lexicon.button.back.info=Umschalttaste drücken und klicken, um zum Hauptmenü zurückzukehren
```

Reply:
Oh... "Home" with THAT sense....